### PR TITLE
Add content for Installation Guides

### LINKED ISSUE #2698: Add FINAL Content for Install Guide - HTML
**User Story:**
As Spark once we get the reviewed content back for the install guides we want to add the FINAL Content for GO Live to NDS

**Notes:**

**AC:**
- Add Final content for HTML Installation Guide
- Add Final content for Angular Installation Guide 
- Add Final content for React Installation Guide 
- Angular Spark Setup
- React Spark Setup

### PR DESCRIPTION
**Pull request by bobwise**
## What does this PR do?
Adds content for these guides to the new docs site:

1. HTML Environment Setup
2. HTML Spark Installation
3. HTML Adding a Component
4. Angular Spark Setup
5. React Spark Setup

fixes #2698

### CODE REVIEW

File: src/hooks/use-installing-spark.js [hunk 1]
```diff
@@ -1,15 +1,25 @@
-import { useStaticQuery, graphql } from 'gatsby';
+import { useStaticQuery, graphql } from "gatsby";
```
Comment (afebbraro) [inline src/hooks/use-installing-spark.js]: I think your editor changed these single quotes to doubles. Should be singles

File: src/pages/installing-spark/angular.mdx [hunk 2]
```diff
@@ -173,7 +174,13 @@ added earlier. If everything's
 working correctly, the text
 should be replaced with an animated spinner.
 
-GIF GOES here
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
```
Comment (afebbraro) [inline src/pages/installing-spark/angular.mdx]:
```suggestion
    class="sprk-u-mbm"
```

File: src/pages/installing-spark.mdx [hunk 3]
```diff
@@ -10,72 +10,26 @@ import {
...
-            element="a"
-            href="/installing-spark/react"
-          >
-            React Installation
-          </SprkButton>
-        </div>
-      </SprkCard>
-    </div>
+  <SprkStackItem additionalClasses='sprk-u-Measure'>
+    <p className="sprk-b-TypeBodyTwo sprk-o-Stack__item">
```
Comment (afebbraro) [inline src/pages/installing-spark.mdx]: Switch to a div and can remove that stack item

File: src/pages/installing-spark/html-add-components.mdx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+4. Navigate to the *Docs* tab at the top of the screen.
+
+5. Scroll down until you see the component variant you would like to include in your site.
+
+6. Click *Show code*.
+
+7. Copy the sample code into `index.html`.
+
+> **For example**: If you’re adding a Submission Spinner Button, you’d scroll down on the Button page to find <a href="https://html.sparkdesignsystem.com/?path=/docs/components-button--spinner" target="_blank">Submission Spinner</a>. Then you’d copy the Submission Spinner Button code that looks like this:
```
Comment (afebbraro) [inline src/pages/installing-spark/html-add-components.mdx]: Dave had us change this to loading from spinner. so this sb url will 404

File: src/pages/installing-spark/html-add-components.mdx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+```
+<button class="sprk-c-Button" data-sprk-spinner="click">
+
```
Comment (afebbraro) [inline src/pages/installing-spark/html-add-components.mdx]: extra line here

File: src/pages/installing-spark/html-add-components.mdx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+Submit
+
```
Comment (afebbraro) [inline src/pages/installing-spark/html-add-components.mdx]: extra line here

File: src/pages/installing-spark/html-add-components.mdx [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+> **Note**: make sure you’re including `main.js` after the HTML elements in `index.html`. The `<script>` line should be at the bottom of the `<body>` tag.
+
+8. You should see the component you just installed with all the correct Spark CSS applied.
+
+> **For example**: If you’ve added a Submission Spinner Button you’ll see something like this…
+
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
```
Comment (afebbraro) [inline src/pages/installing-spark/html-add-components.mdx]:
```suggestion
    class="sprk-u-mbm"
```

File: src/pages/installing-spark/html-environment-setup.mdx [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+  </body>
+</html>
+```
+
+You should now have the initial files for your new site. The folder
+structure should look like this:
+
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
```
Comment (afebbraro) [inline src/pages/installing-spark/html-environment-setup.mdx]:
```suggestion
    class="sprk-u-mbm"
```

File: src/pages/installing-spark/html-environment-setup.mdx [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+    style="margin: auto; display: block;"
+    src={folderStructure}
+    alt="A dist folder containing index.html. A src folder containing index.js. A package.json in the root."/>
+</div>
+
+6. Open `index.html` in a web browser. You should see this:
+
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
```
Comment (afebbraro) [inline src/pages/installing-spark/html-environment-setup.mdx]:
```suggestion
    class="sprk-u-mbm"
```

File: src/pages/installing-spark/html-environment-setup.mdx [hunk 10]
```diff
@@ -0,0 +1,228 @@
...
+    style="margin: auto; display: block;"
+    src={helloSpark}
+    alt="Hello Spark!"/>
+</div>
+
+## Installing Webpack
+
+<a href="https://webpack.js.org/" target="_blank">Webpack</a> is a tool that we’ll use to
+ combine all the JavaScript needed to run Spark into one file.
```
Comment (afebbraro) [inline src/pages/installing-spark/html-environment-setup.mdx]:
```suggestion
combine all the JavaScript needed to run Spark into one file.
```

File: src/pages/installing-spark/html-environment-setup.mdx [hunk 11]
```diff
@@ -0,0 +1,228 @@
...
+npm install webpack webpack-cli --save-dev
+```
+
+> The `--save-dev` flag indicates that these packages are required
+ to compile your website, but are not necessary to run the site
+ after compilation.
+
+8. In your `package.json` file that you created in Step 2, look for
+ the `scripts` section. This is a list of custom commands that npm
+  can run against your package. Add a new one for building the site
```
Comment (afebbraro) [inline src/pages/installing-spark/html-environment-setup.mdx]:
```suggestion
can run against your package. Add a new one for building the site
```

File: src/pages/installing-spark/html-environment-setup.mdx [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+   with Webpack:
```
Comment (afebbraro) [inline src/pages/installing-spark/html-environment-setup.mdx]:
```suggestion
with Webpack:
```

File: src/pages/installing-spark/html-installation.mdx [hunk 13]
```diff
@@ -0,0 +1,148 @@
...
+`main.js` in your solution. The compiled Spark CSS and JavaScript will show
+ here.
+
+``` shell
+npm run build
+```
+
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
```
Comment (afebbraro) [inline src/pages/installing-spark/html-installation.mdx]:
```suggestion
    class="sprk-u-mbm"
```

File: src/pages/installing-spark/html-installation.mdx [hunk 14]
```diff
@@ -0,0 +1,148 @@
...
+</body>
+```
+
+> This attribute is required for some Spark components to correctly handle
+focus changes.
+
+## You did it!
+
+You’re ready to start [adding components](./html-add-components) to
+ your site.
```
Comment (afebbraro) [inline src/pages/installing-spark/html-installation.mdx]:
```suggestion
your site.
```

File: src/pages/installing-spark/html-installation.mdx [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+
+## Additional Topics
+
+* The [Icon Installation Guide](./installing-icons) for importing the Spark SVG icon set
+* The [Font Installation Guide](./installing-fonts) for instructions on using the Rocket Sans font
+
+## Troubleshooting
+
+Here are some issues that you might encounter while setting up Spark and
+ how to fix them:
```
Comment (afebbraro) [inline src/pages/installing-spark/html-installation.mdx]:
```suggestion
how to fix them:
```

File: src/pages/installing-spark/html-installation.mdx [hunk 16]
```diff
@@ -0,0 +1,148 @@
...
+import spark from "@sparkdesignsystem/spark/spark";
+```
+
+---
+
+**Issue: `WARN Hit error self signed certificate in certificate chain`**
+
+This is often caused by corporate firewall configurations and
+can sometimes be fixed by prefixing `npm install` with 
+`NODE_TLS_REJECT_UNAUTHORIZED=0`.
```
Comment (afebbraro) [inline src/pages/installing-spark/html-installation.mdx]: I thought we had decided to not include this in here. But, I forget, could you remind me?

Comment (bobwise) [reply src/pages/installing-spark/html-installation.mdx]: I vaguely remember discussing it but I don't remember deciding anything

Comment (bobwise) [reply src/pages/installing-spark/html-installation.mdx]: I'm fine deleting it if you think that's best

Comment (afebbraro) [reply src/pages/installing-spark/html-installation.mdx]: can you resync up with @yodasw16  on it?

**Pull request merged**